### ISSUE TITLE
[500]: " "guild_id" " not found in match_info

### ISSUE THREAD

**Issue by Drxckzyz** [FerrisChat/Server#60]
Bug Report:
Api returns an Internal Server Error When Deleting Channel

Steps to Reproduce:

Create Channel
Delete created Channel
Code:
Link: https://github.com/FerrisChat/Ferris-lib/blob/main/test/test.js

Error:
```
"guild_id"
" not found in match_info: this is a bug, please report it at \
                    https://github.com/FerrisChat/Server/issues/new?assignees=tazz4843&labels=bug&\
                    template=api_bug_report.yml&title=%5B500%5D%3A+"
"guild_id" "+not+found+in+match_info"
```

**Comment by Drxckzyz** [comment]
Pr was created for this, #61

**Comment by tazz4843** [comment]
closed in #61
